src/components/steps/StepInstitutionType.tsx

```diff
@@ -0,0 +1,96 @@
+import { useTheme } from '@mui/material';
+import { FormControl, FormControlLabel, Grid, Radio, RadioGroup, Typography } from '@mui/material';
+import { Box } from '@mui/system';
+import React from 'react';
+import { useTranslation, Trans } from 'react-i18next';
+import { OrganizationType, StepperStepComponentProps } from '../../../types';
+import { OnboardingStepActions } from '../OnboardingStepActions';
+
+type Props = StepperStepComponentProps & {
+  organizationType: OrganizationType;
+};
+
+const organizationTypeValues = [
+  { labelKey: 'pa', value: 'PA', label: 'Pubblica Amministrazione' },
+  { labelKey: 'gsp', value: 'GSP', label: 'Gestore di servizi pubblici' },
+  { labelKey: 'scp', value: 'SCP', label: 'Società a controllo pubblico' },
+  { labelKey: 'pt', value: 'PT', label: 'Partner tecnologico' },
+];
+
+export default function StepInstitutionType({ back, forward, organizationType }: Props) {
+  const [selectedValue, setSelectedValue] = React.useState<OrganizationType>(organizationType);
+
+  const { t } = useTranslation();
+  const theme = useTheme();
+
+  const handleChange = (value: string) => setSelectedValue(value as OrganizationType);
```

Comment:
After the tipization, this method will accept an OrganizationType parameter, thus this cast will be not more necessary (in this way the solution will be less error prune